yugabyte_cloudformation.yaml

```diff
@@ -36,7 +36,7 @@ Mappings:
       CIDR: "10.0.2.0/24"
   YugaByteConfig:
     Edition: 
-      Value: "ce"
+      Value: "YB"
```

Comment:
We should get rid of edition all together.

Reply:
removed edition dependency.